tests/e2e/cucumber/tagging.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Tagging component
+
+  Background:
+    Given a query tagging API
+    And   an add to cart tagging API
+
+  Scenario Outline: 1. Check some actions trigger the query tagging.
+    Given start page with "<view>" size view
+    When  start button is clicked
+    And   "<query>" is searched
+    Then  query tagging request should be triggered
+    Given a URL with query parameter "<query>" and "<view>" size view
+    Then  query tagging request should be triggered
+    And   results page number 1 is loaded
+    When  scrolls down to next page
+    Then  results page number <pageNumber> is loaded
+    And   query tagging request has been triggered
+    And   page <pageNumber> query tagging request is triggered
```

Comment:
```suggestion
    And   page "<pageNumber>" query tagging request is triggered
```

Reply:
That's not correct, strings and integers are represented in a different way.